src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -1,10 +1,16 @@
 <template>
   <span class="vocab license-icons">
-    <FontAwesomeIcon
-      v-for="(icon, index) in processedIconList"
-      :key="index"
-      :icon="['fab', icon]"
-      fixed-width/>
+    <Popup v-bind="$attrs" :to="popupDirection" :color="popupColor" :action="popupAction" v-if="showPopup">
+      <FontAwesomeIcon v-for="(icon, index) in processedIconList" :key="index" :icon="['fab', icon]" fixed-width />
+      <template #popup>
+        <Paragraph style="font-size: 1.2599rem">
+          {{ popupContent }}
+        </Paragraph>
+      </template>
+    </Popup>
+    <span v-else>
```

Comment:
Rather than adding another level in the DOM tree, use `<template>`, a virtual tag that is not rendered to the DOM.